app/Http/Controllers/ExportController.php

```diff
@@ -213,22 +212,22 @@ public function exportExpansionLoot($expansionSlug, $fileType)
         }
 
         $subdomain = 'www';
-        if ($expansionId == 1) {
+        if ($expansionId === 1) {
             $subdomain = 'classic';
-        } else if ($expansionId == 2) {
+        } else if ($expansionId === 2) {
             $subdomain = 'tbc';
         }
 
-        $locale = '';
-        if (Illuminate\Support\Facades\App::getLocale() != 'en') {
-            if ($subdomain == 'www') {
-                $subdomain = '.' . Illuminate\Support\Facades\App::getLocale() . '.';
-            } else {
-                $locale = Illuminate\Support\Facades\App::getLocale() . '.';
+        $locale = App::getLocale();

```

Comment:
We want the locale to be blank when it's english, so that we don't prefix `en` onto the wowhead links. Not a big deal as I'm sure it still works, but it keeps the URL's nice and clean for everyone using English.